lib/kv-extreme-scale.js

```diff
@@ -2,9 +2,12 @@
 // Node module: loopback-connector-kv-extreme-scale
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
-
 'use strict';
 
+var SG = require('strong-globalize');
+SG.SetRootDir(__dirname + '/..');
+var g = SG();
```

Comment:
Can we move this to `lib/globalize.js` please so that when we decide to add more files to `lib` in the future, then the globalization setup is already there?